tap_zendesk/discover.py

```diff
@@ -20,12 +24,40 @@ def load_shared_schema_refs():
 
     return shared_schema_refs
 
-def discover_streams(client):
+def discover_streams(client, config):
     streams = []
+    error_list = []
     refs = load_shared_schema_refs()
 
+
     for s in STREAMS.values():
-        s = s(client)
+        s = s(client, config)
```

Comment:
@prijendev Can you please give understandable variable name instead of 's'

Reply:
In existing code they have used 's'. Change in name will reflect lot off changes in code as it is used in many place.

Reply:
@prijendev Please do the variable name changes. If it means change will reflect lot off changes that's fine

Reply:
@prijendev Add Comments to the code

Reply:
> @prijendev Add Comments to the code

Added comments to the code. Also, there are more detailed comments in the `try` block just for reference.